### ISSUE TITLE
add search functionality 

### ISSUE THREAD

**Issue by Shreya111111** [Shreya111111/CodeScribe#16]

**Comment by vermaanurag1532** [comment]
please assign it to me i can di this for you

**Comment by mridulsrivastavaa** [comment]
I would like to work on this issue under JWOC .

**Comment by ayesha-2101** [comment]
@Shreya111111 I would like to work on this under JWOC

**Comment by Shreya111111** [comment]
@mridulsrivastavaa @ayesha-2101 u can work on this

**Comment by ayesha-2101** [comment]
@Shreya111111 after cloning I am not able to view the page. Can you please guide me a bit on this 
![Screenshot 2024-01-17 072035](https://github.com/Shreya111111/CodeScribe/assets/112239598/7e6295c5-27ee-4c8c-8b9c-2ac6991a816d)

**Comment by Jenis07** [comment]
can you assign me this issue under JWOC?

**Comment by Shreya111111** [comment]
@mridulsrivastavaa have u done with it

**Comment by mridulsrivastavaa** [comment]
> @mridulsrivastavaa have u done with it

I am sorry. I was down with fever. Can I still work on this ?

**Comment by Shreya111111** [comment]
@mridulsrivastavaa yes u can do it but complete it within two days and upload screenshot and video of your functionality

**Comment by surendar-05** [comment]
@Shreya111111  I would like to work on this under JWOC Assign this to me

**Comment by Shreya111111** [comment]
@surendar-05 go ahead

**Comment by Shreya111111** [comment]
@surendar-05 do it in a day or two plus can u change to grid
